go.mod

```diff
@@ -29,6 +29,4 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-replace github.com/nspcc-dev/neo-go/pkg/interop => ./pkg/interop
```

Comment:
But why do we have this replacement gone?

Reply:
It makes it harder (impossible?) to use our module as a dependency.
Error for `neofs-contract` build.
```
go install: github.com/nspcc-dev/neo-go/...@v0.98.1-pre.0.20220121072805-4a5dff2fff10 (in github.com/nspcc-dev/neo-go@v0.98.1-pre.0.20220121072805-4a5dff2fff10):
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```
Which is understandable given that local path is, well, local to my machine.